broker-core/src/test/java/org/apache/qpid/server/store/BrokerStoreUpgraderAndRecovererTest.java

```diff
@@ -165,6 +167,7 @@ public void testUpgradeVirtualHostWithJDBCStoreAndDefaultPool()
         hostAttributes.put("jdbcBigIntType", "mybigint");
         hostAttributes.put("jdbcBlobType", "myblob");
         hostAttributes.put("jdbcVarbinaryType", "myvarbinary");
+        hostAttributes.put("jdbcTimestampType", "mytimestamp");
```

Comment:
Please undo the change as context variable "qpid.jdbcstore.timestampType" did not exist in previous model versions